test/modules/bradc/printModStuff/foo.good

```diff
@@ -25,14 +25,14 @@ end of module search dirs
   $CHPL_HOME/modules/standard/ChapelEnv.chpl
   $CHPL_HOME/modules/standard/CPtr.chpl
   $CHPL_HOME/modules/standard/HaltWrappers.chpl
+  $CHPL_HOME/modules/standard/Reflection.chpl
```

Comment:
Why did this change happen?

Reply:
This test prints out the files that are parsed when compiling a Chapel program, where the order in which they are parsed depends on the order in which they are `use`d/`import`ed.  I added a new `use Reflection` statement to `ChapelBase.chpl`, which is one of the very earliest modules to be parsed, so it moved the parsing of Reflection earlier in the compilation process than it had been before.

Reply:
Makes sense.  Thanks!